Library/Homebrew/brew.rb

```diff
@@ -88,7 +88,7 @@ def require?(path)
   end
 
   if internal_cmd
-    Homebrew.send cmd.to_s.tr("-", "_").downcase
+    Homebrew::InstallCommand.send cmd.to_s.tr("-", "_").downcase
```

Comment:
@MikeMcQuaid that works and i'll push the changes. But what will be the long term solution to this?